src/Features/Core/Portable/ExternalAccess/UnitTesting/SolutionCrawler/UnitTestingInvocationReasons_Constants.cs

```diff
@@ -13,37 +13,47 @@ namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.SolutionCrawler
         public static readonly UnitTestingInvocationReasons DocumentAdded =
             new(
                 ImmutableHashSet.Create<string>(
+#if false // Not used in unit testing crawling
                                     UnitTestingPredefinedInvocationReasons.DocumentAdded,
                                     UnitTestingPredefinedInvocationReasons.SyntaxChanged,
+#endif
                                     UnitTestingPredefinedInvocationReasons.SemanticChanged));
 
         public static readonly UnitTestingInvocationReasons DocumentRemoved =
             new(
                 ImmutableHashSet.Create<string>(
+#if false // Not used in unit testing crawling
                                     UnitTestingPredefinedInvocationReasons.DocumentRemoved,
                                     UnitTestingPredefinedInvocationReasons.SyntaxChanged,
+#endif
                                     UnitTestingPredefinedInvocationReasons.SemanticChanged,
                                     UnitTestingPredefinedInvocationReasons.HighPriority));
```

Comment:
Is HighPriority also unused?

Reply:
yes, it's still used here:

```
        public static readonly UnitTestingInvocationReasons DocumentRemoved =
            new(
                ImmutableHashSet.Create<string>(
                                    UnitTestingPredefinedInvocationReasons.DocumentRemoved,
                                    UnitTestingPredefinedInvocationReasons.SyntaxChanged,
                                    UnitTestingPredefinedInvocationReasons.SemanticChanged,
                                    UnitTestingPredefinedInvocationReasons.HighPriority));
```

and is then checked in:

```
                    private void CheckHigherPriorityDocument(UnitTestingWorkItem item)
                    {
                        Contract.ThrowIfFalse(item.DocumentId != null);

                        if (!item.InvocationReasons.Contains(UnitTestingPredefinedInvocationReasons.HighPriority))
                        {
                            return;
                        }

                        AddHigherPriorityDocument(item.DocumentId);
                    }
```

so it looks like we fast track a doc removal (presumably to get the data for it removed quickly by a feature like you).

Reply:
Makes sense - perhaps this is worthy of a comment / doc comment on the const?

Reply:
will do!